vita3k/gui/src/settings.cpp

```diff
@@ -590,6 +588,9 @@ static const std::vector<std::string> LIST_SYS_LANG = {
     "Dansk", "Norsk", "Polskis", u8"Português (Brasil)", "English (United Kingdom)", u8"Türkçe"
 };
 
+static const ImU32 ARROW_COLOR = 4294967295; // White
```

Comment:
Better to be in hexcimal

Reply:
i already using this in oher place,this function request ImU32 value only

Reply:
0xFFFFFFFF

Reply:
No i said only using this value no hexa

Reply:
Why no hexdacimal? This is for people to recognise this RGB code immediately without doubt.

Reply:
done